### PR TITLE
Don't mutate parameters.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: pegasus/helpers/section_api_helpers.rb [hunk 1]
```diff
@@ -21,20 +21,19 @@ def self.fetch_user_students(user_id)
   end
 
   def self.create(params)
-    name = params[:name].to_s
-    name = 'New Student' if name.empty?
-
-    params[:gender] = nil unless valid_gender?(params[:gender])
-    params[:birthday] = age_to_birthday(params[:age]) if age_to_birthday(params[:age])
+    name = params[:name].to_s.present? ? params[:name].to_s : 'New Student'
```
Comment (balderdash) [inline pegasus/helpers/section_api_helpers.rb:24]: Can these be written as `name = params[:name].to_s || 'New Student'`, or does that behave differently/violate our style guide?

Comment (ashercodeorg) [inline pegasus/helpers/section_api_helpers.rb:24]: That behaves differently as (for example) `nil.to_s` yields `''` (the empty string) and `'' || 'New Student' yields the empty string. Good thought, though!

Comment (balderdash) [inline pegasus/helpers/section_api_helpers.rb:24]: Got it, thanks!

**Comment by balderdash** [comment]
LGTM

File: pegasus/helpers/section_api_helpers.rb (hunk 1 above)
Comment (ashercodeorg) [inline pegasus/helpers/section_api_helpers.rb:24]: Hmm. Let me look into whether all the casting is even necessary. It wouldn't surprise me if `name = params[:name] || 'New Student'` were possible.

**Pull request merged**